egs/lre07/v2/cmd.sh

```diff
@@ -0,0 +1,28 @@
+# "queue.pl" uses qsub.  The options to it are
+# options to qsub.  If you have GridEngine installed,
+# change this to a queue you have access to.
+# Otherwise, use "run.pl", which will run jobs locally
+# (make sure your --num-jobs options are no more than
+# the number of cpus on your machine.
+
+#a) JHU cluster options
+#export train_cmd="queue.pl -l arch=*64*"
+#export decode_cmd="queue.pl -l arch=*64* -l ram_free=4G,mem_free=4G"
+#export cuda_cmd="..."
+#export mkgraph_cmd="queue.pl -l arch=*64* ram_free=4G,mem_free=4G"
+
+#b) BUT cluster options
+#export train_cmd="queue.pl -q all.q@@blade -l ram_free=1200M,mem_free=1200M"
+#export decode_cmd="queue.pl -q all.q@@blade -l ram_free=1700M,mem_free=1700M"
+#export decodebig_cmd="queue.pl -q all.q@@blade -l ram_free=4G,mem_free=4G"
+#export cuda_cmd="queue.pl -q long.q@@pco203 -l gpu=1"
+#export cuda_cmd="queue.pl -q long.q@pcspeech-gpu"
+#export mkgraph_cmd="queue.pl -q all.q@@servers -l ram_free=4G,mem_free=4G"
+
+#c) run it locally...
+export train_cmd=run.pl
+export decode_cmd=run.pl
+export cuda_cmd=run.pl
+export mkgraph_cmd=run.pl
+
```

Comment:
The way the queue options are handled was changed in this commit: https://github.com/kaldi-asr/kaldi/commit/b3bbc038681b5bcd6be02481785f929b5b99e11a

Basically you need to update the comment and the export statements to be more similar to that? The default commands should include queue.pl in it.